submissions/ireneNakiyingi/HTML/bakery.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>FLUFF BAKES</title>
+        <nav>
+            <a href="bakery.html">Home</a>
+            <a href="cakes.html">Cakes</a>
+            FLUFF BAKES 🎂🍪🍩🥨
+            <a href="bread.html">Bread</a>
+            <a href="contact.html">Contact</a>
```

Comment:
Also, do all these pages exist? You ought to have only two pages for now. The contact page and the product page